workspace.go

```diff
@@ -314,12 +340,30 @@ func (s *workspaces) Read(ctx context.Context, organization, workspace string) (
 		return nil, err
 	}
 
-	w := &Workspace{}
+	w := &Workspace{
+		readme: func(ctx context.Context) (string, error) {
+			ru := u + "?include=readme"
+			r := &workspaceReadme{}
+			req, err := s.client.newRequest("GET", ru, nil)
+			if err != nil {
+				return "", err
+			}
+			err = s.client.do(ctx, req, r)
+			if err != nil {
+				return "", err
+			}
+			return r.Readme, nil
+		},
+	}
 	err = s.client.do(ctx, req, w)
 	if err != nil {
 		return nil, err
 	}
 
+	// durations come over in ms
+	w.ApplyDurationAverage *= (time.Second / time.Millisecond)
+	w.PlanDurationAverage *= (time.Second / time.Millisecond)
```

Comment:
I don't think I understand this, can you elaborate a bit?

Reply:
This was wrong. fixed and I hope it makes more sense now